### PR TITLE
Upgrade alfresco-build-tools to 1.9.2

### PR DESCRIPTION
**Pull request by atchertchian**
Ref: https://github.com/Activiti/Activiti/issues/4029

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -10,7 +10,6 @@ repos:
       - id: helm-lint
       - id: kubepug-minimum
       - id: kubepug-recommended
-      - id: checkov-helm
```
Comment (atchertchian) [inline .pre-commit-config.yaml:13]: Please check that this removal is ok

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -18,10 +18,10 @@ jobs:
   pre-commit:
     runs-on: ubuntu-20.04
     steps:
-      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-helm-docs@v1.9.1
-      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-kubepug@v1.9.1
-      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-checkov@v1.9.1
-      - uses: Alfresco/alfresco-build-tools/.github/actions/pre-commit@v1.9.1
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-helm-docs@v1.9.2
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-kubepug@v1.9.2
+      - uses: bridgecrewio/checkov-action@v12.1708.0
```
Comment (gionn) [inline .github/workflows/main.yml:23]: I see only now that the existing checkov available in pre-commit:
https://github.com/Alfresco/alfresco-build-tools/blob/master/.pre-commit-hooks.yaml#L52

was having some logic regarding which folder to pass while invoking checkov itself, along with --framework kubernetes

at least I would try adding here:
```
with:
  framework: kubernetes
```

Comment (atchertchian) [reply .github/workflows/main.yml:23]: Done, thanks :+1: 
Do you think that would be better on the cloud-full-chart one too?

Comment (atchertchian) [reply .github/workflows/main.yml:23]: As discussed, will add back the equivalent check in pre-commit hooks instead

File: .pre-commit-config.yaml (hunk 1 above)
Comment (atchertchian) [reply .pre-commit-config.yaml:13]: added back

**Comment by atchertchian** [comment]
Added back bridgecrewio/checkov-action usage because the pre-commit hook does not install checkov... (removing commit  https://github.com/Activiti/activiti-cloud-common-chart/pull/49/commits/0f6ccf2df0d31c170c208c2cfea21e5b15a3158b)

**Review by miguelruizdev — APPROVED** [review]

**Review by gionn — APPROVED** [review]

**Pull request merged**